core/src/graphql/runner.rs

```diff
@@ -21,6 +21,10 @@ lazy_static! {
             u64::from_str(&s)
                 .unwrap_or_else(|_| panic!("failed to parse env var GRAPH_GRAPHQL_QUERY_TIMEOUT"))
         ));
+    static ref GRAPHQL_MAX_COMPLEXITY: Option<u64> = env::var("GRAPH_GRAPHQL_MAX_COMPLEXITY")
```

Comment:
While we're at it, should we also make `GRAPH_GRAPHQL_MAX_DEPTH` a variable?

Reply:
Sure.